### ISSUE TITLE
Domínio com Problemas

### ISSUE THREAD

**Issue by mrprompt** [nodebr/NodeBR.com#21]
O domínio nodebr.com está expirado. :(

**Comment by alanhoff** [comment]
Also https://nodebr.com redireciona para femdomsoxtoys.com

**Comment by mrprompt** [comment]
wow.. was working for me. Now, have a SSL error.

**Comment by ayananda** [comment]
Sim,  não renovei ele, tem quase um ano que a comunidade não propôs nada nem ajudou no conteúdo.

**Comment by albertosouza** [comment]
Sobre conteúdo ... a comundiade continua criando conteúdos, cursos e projetos com Node.js. Acho que o que falta é um agregador

**Comment by rafaeldaime** [comment]
Teve durante 2 anos um blog em jekyll aberto a todos. Só tivemos um post
que não foi meu. Por isso abandonei o projeto.

Em Qua, 26 de out de 2016 17:24, Alberto Souza notifications@github.com
escreveu:

**Comment by mrprompt** [comment]
Entendi, passo pela mesma situação em outra comunidade. Mas, se quiserem ajuda para retomar o projeto, estou aqui. ;)
